syntaxes/sway.tmLanguage.json

```diff
@@ -249,6 +249,36 @@
         }
       }
     },
+    {
+      "comment": "Type declaration without name",
+      "begin": "\\b(storage)",
+      "end": "[\\{\\(;]",
+      "beginCaptures": {
+        "1": {
+          "name": "storage.type.sway"
+        }
+      },
+      "patterns": [
+        {
+          "include": "#block_comment"
+        },
+        {
+          "include": "#line_comment"
+        },
+        {
+          "include": "#type_params"
+        },
+        {
+          "include": "#core_types"
+        },
+        {
+          "include": "#pub"
+        },
+        {
+          "include": "#where"
```

Comment:
Just need to remove this and good to go, I think. Deferring to @JoshuaBatty since I'm biased haha

Reply:
Yeah I agree Austin. Happy for this PR to land once the where branch is removed.

Reply:
Dang, I thought I got rid if it already ! will remove shortly.